src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -626,6 +626,9 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 			this._linkManager = this._instantiationService.createInstance(TerminalLinkManager, xterm.raw, this._processManager!);
 			this._areLinksReady = true;
 			this._onLinksReady.fire(this);
+			if ((!isSafari && this._configHelper.config.gpuAcceleration === 'auto') || this._configHelper.config.gpuAcceleration === 'on') {
```

Comment:
waiting for on process ready allows `xterm.raw.element` to become defined, which is necessary to load the webgl add-on

Reply:
can't just call from the constructor (after `xterm.raw` is defined because it's async

Reply:
Immediately after `this.raw.open` is called is ideal

Reply:
that's in a synch method. could create an event, but that seems like overkill

Reply:
https://github.com/microsoft/vscode/blob/bdb513f5451a2045b7dc5daef7bd75dd1ac24135/src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts#L151

Reply:
Calling `xterm.enableWebglRenderer();` without the await should be alright, that's similar to how it's done in the event anyhow as nothing happens after the event resolves.